src/content/docs/logs/forward-logs/stream-logs-using-kinesis-data-firehose.mdx

```diff
@@ -19,6 +19,10 @@ If your log data is already being monitored by [Amazon CloudWatch Logs](https://
 
 Forwarding your CloudWatch Logs or other logs compatible with a Kinesis stream to New Relic will give you enhanced <InlinePopover type="logs"/> capabilities to collect, process, explore, query, and alert on your log data.
 
+<Callout variant="important">
+You should not enable Log Streaming using Kinesis Data Firehose if you're a [FedRAMP customer](/docs/security/security-privacy/compliance/certificates-standards-regulations/fedramp-moderate), because AWS Kinesis Data Firehose integration is not currently FedRAMP authorized.
```

Comment:
```suggestion
If you're a [FedRAMP customer](/docs/security/security-privacy/compliance/certificates-standards-regulations/fedramp-moderate), you should not enable Log Streaming using Kinesis Data Firehose because AWS Kinesis Data Firehose integration is not currently FedRAMP authorized.
```